test/tests/slider_slider-color-viewer.js

```diff
@@ -735,31 +774,6 @@ ariaTest(
     // Send 1 key to blue slider
     const blueSlider = sliders[2];
     await blueSlider.sendKeys(Key.PAGE_DOWN);
-
-    t.is(
```

Comment:
Did you intend to remove these tests? If so, the two lines above them (775-776) should also be removed, since there are no tests checking the result of sending `key.PAGE_DOWN` (though it seems like the tests should probably be re-added for symmetry with red and green).

Reply:
@smhigley 
Thanks for catching this, I added the rest of the test in.